src/components/application_manager/include/application_manager/application_manager_impl.h

```diff
@@ -838,7 +838,7 @@ class ApplicationManagerImpl
     * @return Resume Controller
     */
   resumption::ResumeCtrl& resume_controller() OVERRIDE {
-    return resume_ctrl_;
+    return *resume_ctrl_.get();
```

Comment:
@dev-gh I used here avto_ptr and I don't see point to use SharedPtr because we don't share this pointer.